Packs/Carbon_Black_Enterprise_Live_Response/Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -2719,5 +2719,5 @@ script:
     description: Executes an endpoint memory dump.
   runonce: false
 tests:
-  - CarbonBlackLiveResponseTest
+  - Carbon Black Live Response Test
```

Comment:
did this one ever worked?

Reply:
No idea, planned to see during the build